account_fiscal_position_autodetect_optional_vies/models/account_fiscal_position.py

```diff
@@ -14,32 +14,32 @@ class AccountFiscalPosition(models.Model):
     )
 
     @api.model
-    def search(self, args, offset=0, limit=None, order=None, count=False):
-        if "vat_vies_required" in self.env.context:
-            args = expression.AND(
+    def search(self, domain, offset=0, limit=None, order=None, count=False):
+        if self.env.context.get("vat_vies_required"):
```

Comment:
Previous check was better, we only need to modify the domain if `vat_vies_required` is in the context.

Reply:
Changed!